### DISCUSSION TITLE
When Can We Change The Interval <10 Minutes

### DISCUSSION THREAD

**Discussion by zipperbrave** [AutomaApp/automa#48] [Q&A] [answered]
I want to change the Interval to 5 minutes (as stated in the input box), but it keeps at 10 (Minimum)
When will this update be available so we can change the Interval for less than 5 Minutes?

**Comment by Kholid060** [comment]
I'll lower the min interval to 1 minute in the future update

**Comment by Kholid060** [comment, marked as answer]
You can now change the interval < 10 minutes in v0.4.4
